corehq/apps/reports/filters/devicelog.py

```diff
@@ -10,7 +10,7 @@
 
 class DeviceLogTagFilter(BaseReportFilter):
     slug = "logtag"
-    label = ugettext_noop("Filter Logs by Tag")
+    label = _("Filter Logs by Tag")
```

Comment:
I should have reviewed more closely. this and may other places in this PR like it that used to be `ugettext_noop` should have been changed to `ugettext_lazy`. This is because `ugettext` translates in-place, but since these are top-level/global strings executed when the file is loaded, there's no user context at that point, and thus no way of knowing what language to translate to; `ugettext_lazy`, on the other hand, marks them for translation and then translates them when they are _used_. (`ugettext_noop`, short for "no operation", marks them for tranlsation, but doesn't actually translate them unless you later call `{% trans %}` or some other such utility on them. `ugettext_lazy` is almost always preferable to `ugettext_noop`.) 

For more on `ugettext` vs. `ugettext_lazy` see e.g. http://stackoverflow.com/questions/4160770/when-should-i-use-ugettext-lazy#4164683